dataprofiler/tests/profilers/test_integration_structured_labeler_diff.py

```diff
@@ -0,0 +1,24 @@
+import os
+import unittest
+
+import numpy as np
+import pandas as pd
+
+from dataprofiler import Data, Profiler
+
+test_root_path = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+
+
+class TestStructDiff(unittest.TestCase):
+    def test_diff(self):
+        # divide dataset in half
+        test_dir = os.path.join(test_root_path, "data")
+        data = Data(os.path.join(test_dir, "csv/diamonds.csv"))
```

Comment:
if we can reduce this down when we finalize the PR, that'd be awesome. e.g. no labeler or anything expensive we don't need. even less data maybe if possible

Reply:
totally agree, this is super rough implementation

Reply:
actually included a new test in `test_text_column_profile.py` which is much cleaner, much less data, and re-creates exactly the problem we were trying to solve in the bug